.imptest

```diff
@@ -1,7 +1,15 @@
 {
-  "modelId": "FDmwxg-3sYAx" /* April */,
-  "devices": ["233f4aafaf952dee" /* EI.Test.G */],
-  "agentFile": "iothub.class.nut",
-  "stopOnFailure": false,
-  "timeout": 60
-}
+    "apiKey": "",
```

Comment:
I'm not sure it'll work. Usually we just omit the apiKey properly, which is then taken from the environment. Please make sure it works this way (env variable if set overrides this one). Thanks!